src/EnergyPlus/DataSizing.cc

```diff
@@ -601,6 +608,101 @@ namespace DataSizing {
         this->HeatLoadSeq = this->HeatLoadSeq * ratio;
     }
 
```

Comment:
New function to reset all Data* variables used during sizing. This ensures the next equipment will not inadvertently use previously set variables.